book/website/collaboration/inclusive-events/inclusive-events-schedules.md

```diff
@@ -0,0 +1,286 @@
+For example, people with an experience of cognitive impairment, as well as autistic people, can find unstructured sessions daunting which will negatively impact their ability to fully engage. 
+
+The guidance you provide will be different depending on your event. 
+For example, a public lecture may ask attendees to submit questions for the speaker at the end of the lecture. 
+For this example, indicate in your guidance how long there will be for questions, how to use any online software you may be using to collect audience questions, and also how many questions you are likely going to answer. 
+This can help avoid any feeling attendees may have about being ignored or not heard if their question is not addressed. 
+
+For other events that require active participation, such as a focus group or discussions, consider including guidance on the format discussions will have, and if there are any specific topics or prior material attendees should be familiar with in order to take part. 
+Try to identify if your event will facilitate discussion between attendees, or if the sessions are to be chaired and led by a facilitator. 
+Communicating to attendees that participation is expected, and what the specific format of participation will be, can help alleviate worry or anxiety that attendees may have, and also set the session up for success. 
+
+(cl-inclusive-events-engagement-pathways)=
+### Establishing Pathways for Engagement
+People with lived experience may have varying comfort levels with public events, in addition to concerns about how any disabilities have been supported (or not) in past events.
+Consider providing explicit guidelines and options for engagement to ensure that everyone is able to contribute at their own pace and in a manner that suits their preferences and needs. 
+
+For example, a discussion or contribution-based event may consider offering both spoken and written contributions. 
+"Speaking" group(s) can be facilitated by a note taker, and the "Writing" group(s) can work together in collaborative documents. 
+You may also consider recording your event's content and making documents available online as part of supporting asynchronous 
+contributions from people in multiple time zones or who may experience difficulties in contributing during "live" events. 
+
+Consider designing sessions that support multiple channels of engagement to be as inclusive of as many accessibility requirements and preferences as possible. 
+Because all events are limited by the resources available to plan and host the event, it is important to provide clear guidance on what engagement means for your event and to work with a representative with lived experience to identify what methods of engagement your event will need to support.
```

Comment:
I feel the limited resources heads up might have to go up earlier as that is a general limitation of inclusion/accessibility of events?

Reply:
I added a bit more to the sentence above to help it flow better. 

I'm also happy to take out the first half of the sentence on line 111 - but I feel like it's important to acknowledge that perfect may not be possible based on available resources for this suggestion.